blog/Announcing-Bitcoin-Cash-Node-Japanese.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: layout.html
+---
+
+<% set('title', 'ビットコインキャッシュノード（Bitcoin Cash Node') %>
```

Comment:
The closing ')' in the title is being dropped, I think because there is a real quote at the wrong spot (one character too soon)